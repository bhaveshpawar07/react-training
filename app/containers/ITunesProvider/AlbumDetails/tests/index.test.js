/**
 *
 * Tests for AlbumDetails container
 *
 *
 */

import React from 'react';
// import { fireEvent } from '@testing-library/dom';
import { renderProvider } from '@utils/testUtils';
import { AlbumDetailsTest as AlbumDetails } from '../index';

describe('<AlbumDetails /> container tests', () => {
  let submitSpy;

  beforeEach(() => {
    submitSpy = jest.fn();
  });

  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<AlbumDetails dispatchGetSongDetails={submitSpy} />);
    expect(baseElement).toMatchSnapshot();
  });
});
