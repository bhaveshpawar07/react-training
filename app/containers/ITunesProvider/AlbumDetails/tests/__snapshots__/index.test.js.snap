// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlbumDetails /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="AlbumDetails__Container-sc-1i5tplu-0 dhIudf"
    >
      <div
        class="ant-card ant-card-bordered AlbumDetails__CustomCard-sc-1i5tplu-1 cjiiHu"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="AlbumDetails__CustomDiv-sc-1i5tplu-2 fFJoOb"
          >
            <img
              class="AlbumDetails__CustomImg-sc-1i5tplu-4 ElwFu"
            />
            <div
              class="AlbumDetails__SongDetailsDiv-sc-1i5tplu-3 fOCpGr"
            >
              <p
                class="T__StyledText-gjlic1-0 kYcyRq AlbumDetails__CustomT-sc-1i5tplu-5 eERHWc"
                data-testid="t"
                font-size="20"
                title="Track Name: undefined"
              />
              <p
                class="T__StyledText-gjlic1-0 kYcyRq AlbumDetails__CustomT-sc-1i5tplu-5 cDpuxj"
                data-testid="t"
                font-size="14"
                title="Artist Name: undefined"
              />
              <p
                class="T__StyledText-gjlic1-0 kYcyRq AlbumDetails__CustomT-sc-1i5tplu-5 fTHLBN"
                data-testid="t"
                font-size="12"
                title="Collection Name: undefined"
              />
              <span
                aria-label="play-circle"
                class="anticon anticon-play-circle"
                data-testid="play-song"
                role="img"
                style="font-size: 20px;"
                tabindex="-1"
                title="Listen: undefined"
              >
                <svg
                  aria-hidden="true"
                  data-icon="play-circle"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm144.1 454.9L437.7 677.8a8.02 8.02 0 01-12.7-6.5V353.7a8 8 0 0112.7-6.5L656.1 506a7.9 7.9 0 010 12.9z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
